Dockerfile

```diff
@@ -14,49 +14,49 @@ ENV DEBIAN_FRONTEND="noninteractive" \
 RUN apt-get update \
   && apt-get upgrade -y \
   && apt-get install -y --no-install-recommends \
-    build-essential \
```

Comment:
I'm guess my formatter in vscode did a few ticks here...

Reply:
I don't feel strongly about this 🤷

Reply:
Sorry - some unrelated changes sneaked in.

I did an update to remove them